examples/03_connectivity/plot_signal_extraction.py

```diff
@@ -90,4 +94,53 @@
 plt.subplots_adjust(left=.01, bottom=.3, top=.99, right=.62)
 plt.suptitle('No confounds', size=27)
 
-plt.show()
+###############################################################################
```

Comment:
Maybe it would be useful to use a few restructured-text titles, to give a bit more vertical space to the example.